### ISSUE TITLE
Support ARIA role for accessibility on Map component

### ISSUE THREAD

**Issue by Scaralu** [PaulLeCam/react-leaflet#1009]
## Problem
Following WCAG 1.3.1 rules, for a component to be accessible, it's needed to ensure that elements in the focus order have an appropriate role.

## Expected behavior
The leaflet container should have an accessible and meaningful role.

## Reproduce steps
Render a map, and check its accessibility with axe/wave;

**Comment by Scaralu** [comment]
I'm going to start working on it.
Once I have something done, I'll post updates here.

**Comment by gitkumi** [comment]
Would also be nice if we can control the `tab-index` of `MapContainer` and `Marker`. Tabbing through all those `Marker` can be tedious.

**Comment by Leningram** [comment]
Moreover, if I have a map absolutely positioned and it goes out of the screen for a bit, when I click on it, the document screen moves to show all the map. It occurs because the map container has tabIndex=1, when I remove it - I can click on the map without having my screen moving.
